core/core_bind.h

```diff
@@ -63,9 +63,11 @@ class ResourceLoader : public Object {
 	};
 
 	enum CacheMode {
-		CACHE_MODE_IGNORE, // Resource and subresources do not use path cache, no path is set into resource.
-		CACHE_MODE_REUSE, // Resource and subresources use patch cache, reuse existing loaded resources instead of loading from disk when available.
-		CACHE_MODE_REPLACE, // Resource and subresource use path cache, but replace existing loaded resources when available with information from disk.
+		CACHE_MODE_IGNORE,
+		CACHE_MODE_REUSE,
+		CACHE_MODE_REPLACE,
```

Comment:
Unless the prior comments were completely wrong I don't see why removing them outright.

Reply:
I removed them to keep the docs as the only source of truth. Well, actually two sources of truth, because due to how this enum is doubly-exposed, they have to be repeated. I had the impression that, when there's a natural place in the docs for some item, the unwritten rule is to document there instead of the source code. I can understand some will check the C++ enum, but if we keep explanations there, we'll already have three copies. Sounds like it's too easy for them to get out-of-sync over maintenance.

Reply:
That's true, but for very "deep" and somewhat important constants such as these, it's very unlikely their behavior "desyncs" with the documentation unnoticed.

But at the same time you _did have_ to make this PR in part because of that, so I can understand.

Reply:
Let's see if we can get more opinions for the scales to tip one way or the other.

Reply:
The current documentation was copied from these comments. I think it's fine to remove them now.